tools/infrasys/src/main.rs

```diff
@@ -0,0 +1,767 @@
+use async_trait::async_trait;
+use log::{debug, info, trace, warn};
+use pubsys_config::{InfraConfig, KMSKeyConfig, SigningKeyConfig};
+use rusoto_cloudformation::*;
+use rusoto_core::Region;
+use rusoto_s3::*;
+use shell_words::split;
+use snafu::{OptionExt, ResultExt};
+use std::collections::HashMap;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::process::Command;
+use std::str::FromStr;
+use std::{env, fs, process, thread, time};
+use structopt::StructOpt;
+use tokio::runtime::Runtime;
+use url::Url;
+
+// TODO: Add Logging throughout so the user knows like "waiting for x resource"
+// COMMENT KEYWORDS: TODO, NOTE, QUESTION, IN-FUTURE
```

Comment:
Is there a difference between `TODO` and `IN-FUTURE`?

Reply:
TODO are things I hope to address asap and IN-FUTURE are some stretch goals / lines to signify where would be a good place to add new features (like using an existing bucket or something)